### DISCUSSION TITLE
No pausable Queues?

### DISCUSSION THREAD

**Discussion by leoamigood** [ixti/sidekiq-throttled#136] [Q&A]
Thank you for maintaining this gem. 

Just curious why the pausable queues functionality [was removed](https://github.com/ixti/sidekiq-throttled/pull/9)?

I thought of using it..

**Comment by ixti** [comment]
@leoamigood it was done in attempt to simplify the surface of the gem. 0.X branch will have pausable queues as is. Future 1.X release will have it too, but I'm not yet certain on how that will be implemented. :))

**Comment by ixti** [comment]
In order to bring pausible queues, starting with v1.0.0 (soon to be released), you should add to your Gemfile:

``` ruby
gem "sidekiq-pauzer", "~> 4.0"
```

Then in your sidekiq initialization code add:

``` ruby
require "sidekiq"
require "sidekiq/pauzer"

require "sidekiq/web"
require "sidekiq/pauzer/web"
```
